### DISCUSSION TITLE
How do I add navigation arrows for iframed revealjs presentations?

### DISCUSSION THREAD

**Discussion by RaymondBalise** [quarto-dev/quarto-cli#8803] [Q&A] [answered]
### Description

The [quarto revealjs website](https://quarto.org/docs/presentations/revealjs/) shows demo slides. They render with navigation controls on the right and left.  How can I add those controls to my web pages that have embedded revealjs slides 

```{=html}
<iframe class="slide-deck" src="make_a_slide_deck/"></iframe>
```

in make_a_slide_deck/ I have

```
---
format: 
  revealjs:
    slide-number: true
    chalkboard: 
      buttons: false
    preview-links: auto
    css: styles.css
---

# Section One

... words can go here ...

## Subsection

... content goes here ...
```

**Comment by cderv** [comment, marked as answer]
This should happen by default following the controls feature which is `controls: auto` by default. From https://quarto.org/docs/presentations/revealjs/presenting.html#navigation-options

> `controls` : Show arrow controls for navigating through slides (defaults to auto, which will show controls when vertical slides are present or when the deck is embedded in an iframe).

This should automatically adds the controls when you host  the website and the iframe is loaded for the presentation. They won't show on localhost - only when deployed. 

This is implemented as a Revealjs plugin
https://github.com/quarto-dev/quarto-cli/blob/1cf5ce21962045d8e7cf5b3c0cf4e15a789ee708/src/resources/formats/revealjs/plugins/support/support.js#L21-L35

Hope it helps
